src/containers/Review/ReviewSectionAssignment.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Segment } from 'semantic-ui-react'
+import { NoMatch, Loading } from '../../components'
+import useGetFullReviewStructure from '../../utils/hooks/useGetFullReviewStructure'
+import { AssignmentDetailsNEW, FullStructure } from '../../utils/types'
+
+// Component renders a line for review assignment within a section
+// to be used in ReviewHome and Expansions
+const ReviewSectionAssignment: React.FC<{
+  sectionId: number
+  assignment: AssignmentDetailsNEW
+  fullApplicationStructure: FullStructure
+}> = ({ sectionId, assignment, fullApplicationStructure }) => {
```

Comment:
```suggestion
interface ReviewRowProps {
  sectionId: number
  assignment: AssignmentDetailsNEW
  fullApplicationStructure: FullStructure
}
const ReviewRow: React.FC<ReviewRowProps> = ({ sectionId, assignment, fullApplicationStructure }) => {
```

With suggestion to rename ReviewSectionAssignment to ReviewRow

Reply:
nice, i am doing this in the upcoming PR. Do you have a suggestion for a better name for ReviewSectionAssignment ? I've renamed in upstream PR to ReviewSectionSummary, but it's not descriptive enough. In verbose form its: ReviewSectionAssignmentSummaryRow